bot_modules/search_commands/tags.js

```diff
@@ -0,0 +1,137 @@
+const stringSimilarity = require('string-similarity')
+const Discord = require('discord.js')
+const Utils = require('../../utils')
+
+/**
+ * Handle the tag command
+ *
+ * @param {Discord.Message} msg The original message send by the user
+ * @param {Array} args The list of arguments passed to the command
+ */
+exports.handleTagCommand = async (msg, args) => {
+  const embed = new Discord.MessageEmbed()
+
+  // Check to make sure we have a tag
+  if (args.length <= 1) {
+    embed.setTitle('Invalid usage')
+    embed.setDescription('Missing tag to check. `!tag <tag>`')
+    embed.setColor(0xff0000)
+    msg.channel.send(embed)
+    return
+  }
+
+  // Remove the first argument (the command)
+  args.splice(0, 1)
+
+  // Join the other arguments together and collect the tags
+  const query = args.join(' ').trim()
+  const tags = await getTags()
+  let foundTag = false
+
+  // Search the tags by an exact starting match
+  tags.forEach((tag) => {
+    if (tag.name.toLowerCase().startsWith(query.toLowerCase())) {
+      foundTag = true
+
+      sendTagEmbed(msg, tag)
+    }
+  })
+
+  // If we haven't found a tag yet use an
+  // algorythm to check for the most similar tag
```

Comment:
This is misspelled...

Reply:
Also should we use string similarity or only accept exact matches?

Reply:
Only exact matches makes sense for tags